README.md

```diff
@@ -139,12 +224,21 @@ spec:
 
 ### Run code checks and unit tests
 
-`make test`
+Run `make test`
 
 ### Run e2e tests
 
 1. Deploy the operator as explained above
-2. Run `make test-e2e`
+2. (Only for AWS platforms) Run `make ocp-aws-credentials` to add sufficient [CredentialsRequest](https://github.com/medik8s/fence-agents-remediation/blob/main/config/ocp_aws/fence_aws_credentials_request.yaml).
+3. Export the operator installed namespace (e.g., *openshift-operators*) and run `export OPERATOR_NS=INSTALLED_NAMESPACE && make test-e2e`
```

Comment:
Should this be `openshift-workload-availability` now?

Reply:
Moreover, I wonder if `export OPERATOR_NS=INSTALLED_NAMESPACE` can be misunderstood. To play safe(r), we could write something like

Export the operator installed ... before running the e2e test:

```
export OPERATOR_NS=openshift-workload-availability && make test-e2e
```